server/src/repositories/search.repository.ts

```diff
@@ -241,6 +262,7 @@ export class SearchRepository {
     return this.db.transaction().execute(async (trx) => {
       await sql`set local vchordrq.probes = ${sql.lit(probes[VectorIndex.CLIP])}`.execute(trx);
       const items = await searchAssetBuilder(trx, options)
+        .selectAll('assets')
```

Comment:
Do you actually need to select *all* here?

Reply:
This is unchanged from the previous code, just moved. You can see that `selectAll` was called on line 299 in database.ts.